### PR TITLE
[TASK] Layout UI fixes

### PR DESCRIPTION
**Pull request by vhpoet**

### CODE REVIEW

File: src/templates/tabs/settingsgateway.jade [hunk 1]
```diff
@@ -1,17 +1,20 @@
 section.col-xs-12.content(ng-controller="SettingsGatewayCtrl")
   div(ng-show="!loadingAccount && !account.Balance && loadState.account && connected && !debug")
     include ../tabs/banner/_unfunded
+
+  .notification-wrapper
+    .alert.alert-success(ng-show="success.bridge", l10n) Your bitcoin bridge has been changed successfully.
+    .alert.alert-success(ng-show="success.advanced_feature_switch", l10n) Trustline advanced settings have been changed successfully.
```
Comment (mrajvanshy) [inline src/templates/tabs/settingsgateway.jade]: Shouldn't have this notification anymore

File: src/less/ripple/content.less [hunk 2]
```diff
@@ -335,9 +335,8 @@ h1 {
 
 // Popover
 .popover {
-  max-width: none;
```
Comment (MatthewPhinney) [inline src/less/ripple/content.less]: Can you confirm the popover still contains the full Ripple address on the trust page?

**Comment by vhpoet** [comment]
Fixed @mrajvanshy @MatthewPhinney !

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**